src/mui-setup.js

```diff
@@ -12,7 +12,16 @@ export default function muiSetup(sproutBase) {
     sproutBase.overrides.MuiTableCell.root.height = 'auto';
     sproutBase.overrides.MuiTableCell.root.lineHeight = '130%';
     sproutBase.overrides.MuiTableCell.head.height = 'auto';
-    sproutBase.overrides.MuiTableCell.head.lineHeight = '150%';
+    sproutBase.overrides.MuiTableCell.head = {
+      lineHeight: '150%',
+      position: '-webkit-sticky' /* Safari */,
+      // eslint-disable-next-line no-dupe-keys
+      position: 'sticky',
```

Comment:
isn't it just overridden?

Reply:
not really learning from https://developer.mozilla.org/en-US/docs/Web/CSS/position#sticky_positioning

Reply:
Ahh, it is complaining bc this is a js object, not a css definition. Seems to work fine in safari